pom.xml

```diff
@@ -249,5 +249,17 @@
             <type>jar</type>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.ant</groupId>
+            <artifactId>ant</artifactId>
+            <version>1.9.2</version>
```

Comment:
Ant v1.9.6 is the current version. If there is no particular reason to stick with v1.9.3, I would reference the latest release.